README.md

```diff
@@ -314,12 +321,35 @@ Now let's publish the package to artifactory. Run the following command.
 npm publish --registry=https://artifactory.corp.adobe.com:443/artifactory/api/npm/npm-spectrum-release-local
 ```
 
-Now that the package is published, let's revert the changes we did for artifactory by running the following commands.
+Now that the package is published, let's revert the changes we did for artifactory by running the following command.
 
 ```
 git checkout package.json package-lock.json
-rm -rf node_modules && npm install
 ```
 
+Now let's deploy the docs. We need to build the repo, make a copy of `dist` named after the release, checkout the `gh-pages` branch, add the new version of the docs and update references in the redirect files and commit. Run the following commands.
+
+```
+npm run build
+cp -r dist 2.7.2 (replace this with the version you are releasing)
+git checkout gh-pages
+git add 2.7.2 (replace this with the version you are releasing)
+```
+Manually update the old versions in `index.html` and `icons/index.html`. 
```

Comment:
Good info! Yeah, I'd love to use the gh-pages module for this. But I don't think it will handle multiple versions well. Or at least I haven't done it with that module before. I will look into that when I go about automating these steps.